hummingbot/connector/exchange/loopring/loopring_user_stream_tracker.py

```diff
@@ -29,18 +25,18 @@ def logger(cls) -> HummingbotLogger:
     def __init__(self,
                  orderbook_tracker_data_source: LoopringAPIOrderBookDataSource,
                  loopring_auth: LoopringAuth):
-        super().__init__()
-        self._ev_loop: asyncio.events.AbstractEventLoop = asyncio.get_event_loop()
-        self._data_source: Optional[UserStreamTrackerDataSource] = None
-        self._user_stream_tracking_task: Optional[asyncio.Task] = None
         self._orderbook_tracker_data_source = orderbook_tracker_data_source
         self._loopring_auth: LoopringAuth = loopring_auth
+        super().__init__(data_source=LoopringAPIUserStreamDataSource(
```

Comment:
As discussed in the last stand up meeting, we need to do this in this case to instantiate the data source required by super().__init__. The class should be removed in future cleanups anyways, so we are accepting this situation for the moment, to avoid introducing big changes in all connectors with this PR intended for Kucoin.